crates/ruff_linter/src/rules/pandas_vet/rules/assignment_to_df.rs

```diff
@@ -4,9 +4,9 @@ use ruff_text_size::Ranged;
 
 use crate::{Violation, checkers::ast::Checker};
 
-/// ## Deprecated
+/// ## Removed
 ///
-/// This rule has been deprecated as it's highly opinionated and overly strict in most cases.
+/// This rule has been Removed.
```

Comment:
```suggestion
/// This rule has been removed as it's highly opinionated and overly strict in most cases.
```

I think the rest of the sentence is still valuable enough.